requirements.txt

```diff
@@ -1,3 +1,4 @@
+sudo apt-get install libxml2-dev libxslt-dev python-dev zlib1g-dev
```

Comment:
Put this line in the readme.md file. This file is ran by pip so it goes through line by line and looks at the package names and versions to download...it won't work with this first line in it. It'll be essential to add to the readme.md though.